snap/snapcraft.yaml

```
@@ -0,0 +1,52 @@
+name: openstack-network-agents
+summary: OVS provider-bridge helper for OpenStack Network role
+description: |
+  Provides a helper to configure a provider bridge and OVN physnet mapping.
+  This snap is meant to be driven by a subordinate charm and does not ship
+  neutron-ovn-metadata-agent.
+base: core24
+version: "0.1"
+license: Apache-2.0
+grade: devel
+confinement: strict
+
+plugs:
+  network-control:
+    interface: network-control
+  network:
+    interface: network
+  ovn-chassis:
+    interface: ovn-chassis
+  microovn-ovsdb:
+    interface: system-files
+    write:
+      - /var/snap/microovn/common/run/switch
+      - /var/snap/microovn/common/run/switch/db.sock
```

What does this new plug bring that is not shared by ovn-chassis https://github.com/canonical/microovn/blob/72b93b5329ff5b070685d818943b868fe2423f89/snap/snapcraft.yaml#L25C3-L25C14 ?

I believe the definition should look like:

```suggestion
  ovn-chassis:
    interface: content
    target: $SNAP_DATA/ovn
```

(to be checked)

---

I'm a bit surprised you don't need the certificates?

---

Thank you, indeed this fixes the problems regarding accessing the db... Removed the others in favor of this one